Libs/MRML/DisplayableManager/vtkMRMLInteractionWidgetRepresentation.cxx

```diff
@@ -971,7 +971,13 @@ vtkMRMLInteractionWidgetRepresentation::InteractionPipeline::InteractionPipeline
   this->Mapper3D->SetLookupTable(this->ColorTable);
   this->Mapper3D->ScalarVisibilityOn();
   this->Mapper3D->UseLookupTableScalarRangeOn();
+
+  // Update offset so that the handles are always rendered on top of other actors.
+  double offsetFactor = 1.0;
+  double offsetUnits = -2500.0;
```

Comment:
This is very simple, which is good; but hardcoding specific values at many places may become hard to maintain.

What do you think about specifying coincident topology offsets and renderer layer indices at one place, somewhere in MRML Core or DisplayableManager? It should be easily accessible from anywhere, probably with a static function. It could be added for example to `vtkMRMLAbstractDisplayableManager.h`, `vtkMRMLAbstractWidgetRepresentation.h`, `vtkMRML.h`.

Reply:
I agree with the idea of centralizing these and not hard coding anything.  Also I'd like to see getter/setter methods since the values may depend on the hardware/software used in rendering or the datatype of the framebuffer.